tests/unittests/azure_namespace_import/azure_namespace_import.py

```diff
@@ -0,0 +1,50 @@
+import os
+import sys
+import shutil
+import asyncio
+
+from azure.functions_worker import protos
+from azure.functions_worker import testutils
+
+
+async def vertify_nested_namespace_import():
+    test_env = {}
+    request = protos.FunctionEnvironmentReloadRequest(
+        environment_variables=test_env)
+
+    request_msg = protos.StreamingMessage(
+        request_id='0',
+        function_environment_reload_request=request)
+
+    disp = testutils.create_dummy_dispatcher()
+
+    # Mock intepreter starts in placeholder mode
+    import azure.module_a as mod_a  # noqa: F401
+
+    # Mock function specialization, load customer's libraries and functionapps
+    test_root = sys.argv[2]
```

Comment:
Can we rename this to `ns_root` or something to indicate that it is the directory with the second namespace? `test_root` is a term that we already use heavily and can mean a number of things.